posthog/models/feature_flag.py

```diff
@@ -130,12 +138,18 @@ def get_filters(self):
         else:
             # :TRICKY: Keep this backwards compatible.
             #   We don't want to migrate to avoid /decide endpoint downtime until this code has been deployed
-            return {
+            response = {
```

Comment:
You don't need to add this, since it's for the backwards compatible case. Makes it more confusing 😅 .

This is for the case where the `groups` key is not in filters, and there's a direct `properties` key instead. There's no way for the payload to be here.

We might be able to remove this completely, depending on if any old clients remain.

Reply:
There's a related failing test here. Checking again

Reply:
Last valid such flag was in Jan 2021: https://metabase.posthog.net/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7ImRhdGFiYXNlIjozLCJ0eXBlIjoicXVlcnkiLCJxdWVyeSI6eyJzb3VyY2UtdGFibGUiOjQ3LCJmaWx0ZXIiOlsiYW5kIixbImlzLW51bGwiLFsiZmllbGQiLDIzODE5LG51bGxdXV0sIm9yZGVyLWJ5IjpbWyJhc2MiLFsiZmllbGQiLDM4OCxudWxsXV1dfX0sImRpc3BsYXkiOiJ0YWJsZSIsInZpc3VhbGl6YXRpb25fc2V0dGluZ3MiOnt9LCJvcmlnaW5hbF9jYXJkX2lkIjpudWxsfQ==

There does seem to be a lot of trash created too though, where filters are completely empty. Should guard against that, I'll add that in a follow up